Core/Config.cpp

```diff
@@ -1835,10 +1805,6 @@ void Config::RestoreDefaults(RestoreSettingsBits whatToRestore) {
 			ClearRecentIsos();
 			currentDirectory = defaultCurrentDirectory;
 		}
-
-		if (whatToRestore & (RestoreSettingsBits::SETTINGS | RestoreSettingsBits::CONTROLS)) {
-			Load();
```

Comment:
Still can't test, but this should probably be done if we nuke the "controller ini" right? Truth to be told we have a reset for this in control mapping screen, it's probably better to use that path here too.

PS: might need to call save after all of this just to be sure?

Reply:
Seems to work just fine without this, something else is taking care of it somehow. I did test this time :)